src/dropdown/index.jsx

```diff
@@ -271,13 +248,15 @@ const Dropdown = (props) => {
         isActive={show === true}
         onClick={showDropDown}
         hasData={data.hasData}
+        size={props.size || 'M'}
+        background={props.background}
       >
-        <Left>{data.content}</Left>
-        <Right>
-          <Icon position={'right'}>
-            <Arrow rotate={90} />
-          </Icon>
-        </Right>
+        {props.icon && <Icon>{props.icon}</Icon>}
```

Comment:
update definition now that we accept icon

Reply:
didn't understood, do you mean to remove condition.
If yes, there are some places where we don't have icon (like in global app filter, add a page in personal)

Reply:
you need to updated definition file `src/typings/index.d.ts` so that it accepts icon prop in typescript

Reply:
okay

Reply:
done